.github/workflows/build.yml

```diff
@@ -131,7 +117,7 @@ jobs:
         with:
           name: juno-ubuntu-latest
           path: release/linux
-      
+
       - name: Rename Artifacts
```

Comment:
Again, this was originally written for rust. Feel free to start from scratch and rewrite it as long as the basic structure is maintained